Event Delegation/app.js

```diff
@@ -0,0 +1,9 @@
+let inputItems = document.querySelectorAll('input');
+
+for (let i = 0; i < inputItems.length; i++) {
+	inputItems[i].addEventListener('click', (e) => {
+		if (e.target.tagName === 'INPUT') {
+			e.target.style.backgroundColor = 'rgb(255, 255, 0)';
+		}
```

Comment:
Will there be an issue if we remove the `if` statement? And just run `e.target.style.backgroundColor = 'rgb(255, 255, 0)';` right away?